finalProject/tenSpeckledFrogs.js

```diff
@@ -0,0 +1,33 @@
+/*
+
+> 3 speckled frogs sat on a log 
+> eating some most delicious bugs.  
+> One jumped in the pool where its nice and cool,
+> then there were 2 speckled frogs.
+>
+> 2 speckled frogs sat on a log 
+> eating some most delicious bugs.  
+> One jumped in the pool where its nice and cool,  
+> then there was 1 speckled frog.
+>
+> 1 speckled frog sat on a log 
+> eating some most delicious bugs.  
+> One jumped in the pool where its nice and cool,  
+> then there were no more speckled frogs!
+
+## Required
+
+Make your program print the rhyme above for *10* frogs, with attention to where language changes. Notice the very small differences between what should be printed out when 3, 2 or 1 frog is sitting on a log.
+*/
+change1 = 10
+change2 = 9
+change3 = 8
+var poem = [`${change1} speckled frogs sat on a log eating some most delicous bugs. One jumped in the pool where its nice and coool, then there were ${change2} speckled frogs.
```

Comment:
Why did you decide on the array data type for the variable `poem` here?

Reply:
I'm not too sure about string interpolation and maybe the examples i was working with may have had it in one of their examples so i just kept with it? Looking back at it now i can tell it wasn't needed.